test/e2e_extended_tests.sh

```diff
@@ -45,7 +45,7 @@ fi
 export E2E_FUNC_BIN_PATH=$(pwd)/func
 
 go clean -testcache
-go test -v -test.v -tags="e2e" ./test/e2e/
+go test -v -test.v -test.timeout=60m -tags="e2e" ./test/e2e/
```

Comment:
I guess it does take a long time for these tests to run. One hour is a long timeout though. 🤷

Reply:
Well.. actually it takes around 15 to 20min most... I think I overestimated it (for safe)... maybe I can decrease it to 30m(?)..